tequilapi_spec.json

```diff
@@ -7,24 +7,22 @@
   ],
   "swagger": "2.0",
   "info": {
-    "description": "API description.",
-    "title": "Tequila API.",
-    "version": "0.0.1"
+    "description": "The purpose of this documentation is to provide developers an insight of how to\ninteract with Mysterium Node via Tequila API.\nThis should demonstrate all the possible API calls with described parameters and responses.",
```

Comment:
I would not commit this file, and run generation in CI as soon as possible

Reply:
will not commit this file